scripts/parse-contributors.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from pyosmeta.contributors import ProcessContributors
+
+# API_TOKEN = get_api_token()
+API_TOKEN = os.environ.get("GITHUB_TOKEN")
+
+json_files = [
+    "https://raw.githubusercontent.com/pyOpenSci/python-package-guide/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/software-peer-review/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/pyopensci.github.io/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/software-review/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/update-web-metadata/main/.all-contributorsrc",
```

Comment:
Minor side comment, but is there a reason that all these files are missing their extensions? A lot of helpful tooling and infra (pre-commit, editors, many things on GitHub, etc.) doesn't work right if the files don't have the correct extensions.

Reply:
good point - this is out of my control - https://github.com/pyOpenSci/pyopensci.github.io/blob/main/.all-contributorsrc the [all contributors bot](https://allcontributors.org/docs/en/overview) that i use to track contribs across our repos creates this file. so for whatever reason it doesn't have an extension. i could look into whether this has changed since i installed it but i didn't pick that file name. (or atleast i dont think i did?)

Reply:
Oh, I see—this is indeed out of your control, then!

I'd mark this as resolved, except I can't as I'm not an org member. For some strange reason, GitHub's permissions are such that as a non-org-member I cannot mark my _own_ review comments as resolved, but I can mark _other people's_ review comments as resolved on my own PRs, which is exactly backward to what would make sense.